uber/configspec.ini

```diff
@@ -58,6 +58,12 @@ badge_promo_codes_enabled = boolean(default=False)
 # be omitted entirely during preregistration.
 prereg_request_hotel_info_duration = integer(default=0)
 
+# This will be used as the sender for the hotel booking info email.
+prereg_hotel_info_email_sender = string(default='Do Not Reply <noreply@magfest.org>')
+
+# This will be used as the signature for the hotel booking info email.
+prereg_hotel_info_email_signature = string(default='MAGFest')
```

Comment:
I'm okay with this either way, but do you think we even need a signature for this email? Normally those aren't even added unless the sender is listed as a specific person (like the head of guest relations, or the registration department head). If it's really just gonna say "MAGFest," I wouldn't bother.

On the other hand, you already did all the work, so since it's more work to undo it I'd leave it as-is!